### PR TITLE
Create Helm Chart for benchmark-operator

### PR DESCRIPTION
**Pull request by whitleykeith**
This PR creates a helm chart for the benchmark-operator, which can be used to provide an easier installation experience compared to the current process.

With this chart, installing the operator only requires the following:

```bash
oc adm policy -n my-ripsaw add-scc-to-user privileged -z benchmark-operator
helm install ripsaw $CHART_PATH -n my-ripsaw --create-namespace
```

deleting the operator is also simple, requiring the following:

```bash
helm delete ripsaw -n my-ripsaw --purge
```

It's also much easier to parameterize things through the use of Helm Values. For example, changing the operator image and tag is done in the same line as the install:

```bash
cd charts/benchmark-operator
helm install ripsaw . -n my-ripsaw --create-namespace --set operator.image.repository=$OPERATOR_REPO,operator.image.tag=$OPERATOR_TAG
```

### CODE REVIEW

**Review by jtaleric — CHANGES_REQUESTED** [review]
I am not too familiar with Helm, but this seems to be missing a readme and test for the Helm install... i wonder if we could add a simple test to install via the helm chart?

**Comment by whitleykeith** [comment]
@jtaleric I added a README for the helm chart, but I'm not sure how to handle adding a test with the E2E refactoring pending since it would create a good bit of conflicts

File: charts/benchmark-operator/templates/operator.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        name: {{ include "benchmark-operator.fullname" . }}
+    spec:
+      serviceAccountName: {{ include "benchmark-operator.fullname" . }}
+      containers:
+        - name: ansible
+          command:
+          - /usr/local/bin/ao-logs
+          - /tmp/ansible-operator/runner
+          - stdout
+          image: "{{ .Values.operator.image.repository }}:{{ .Values.operator.image.tag | default "master" }}"
```
Comment (rsevilla87) [inline charts/benchmark-operator/templates/operator.yaml:23]: Why don't you default this variable in the values.yaml file?

File: charts/benchmark-operator/templates/serivce-account.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (rsevilla87) [inline charts/benchmark-operator/templates/serivce-account.yaml:1]: s/serivce-account.yaml/service-account.yaml

**Review by rsevilla87 — CHANGES_REQUESTED** [review]
Just a couple of small nits

File: README.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
...
+## Installation
+
+Installing the benchmark-operator is easiest by using the helm chart and can be done with the following commands.
+
+> Note: If running on openshift you'll need to run this command before installing the chart. `oc adm policy -n my-ripsaw add-scc-to-user privileged -z benchmark-operator`
+
+```bash
+git clone https://github.com/cloud-bulldozer/benchmark-operator
+cd benchmark-operator/charts/benchmark-operator
+helm install benchmark-operator . -n my-ripsaw --create-namespace
```
Comment (chaitanyaenr) [inline README.md:15]: We might want to mention that helm needs to be installed on the box and may be also include the helm command to delete the operator that you have in the commit message.

**Review by chaitanyaenr — COMMENTED** [review]
Minor nit, otherwise LGTM.

**Review by rsevilla87 — APPROVED** [review]
LGTM!, nice work @whitleykeith

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**